quickstarts/bot'o'lytics/config.yml

```diff
@@ -0,0 +1,33 @@
+id:
+slug: bot-o-lytics
+title: Bot'O'lytics
+description: |
+  ## Bot'O'lytics
+
+  On every well used website there are not only real users, but also many many bots. Those can be legitimate bots, like search bots, synthetic checks or other automated calls to your service. But even legitimate bots can distort your service's vitals in a way that they don't anylonger represent your real customer's experience.
+  Being able to distinguish real traffic from bot traffic is cruitial in various cases, for example when examining a marketing campain or when analysing your services vitals or service levels.
+
+  ### How to use Bot'o'lytics
+
+  This Quickstart works on top of your APM data. We have included the most common bots, but you can and should customize the dashboard with bots specific to your service.
+
+summary: |
+  On every well used website there are not only real users, but also many many bots. Those can be legitimate bots, like search bots, synthetic checks or other automated calls to your service. But even legitimate bots can distort your service's vitals in a way that they don't anylonger represent your real customer's experience.
+  Being able to distinguish real traffic from bot traffic is cruitial in various cases, for example when examining a marketing campain or when analysing your services vitals or service levels. The Bot'o'lytic Dashboard enables you to analyse your traffic for bot traffic.
+level: Verified
+authors:
+  - Heiko Specht (New Relic)
+keywords:
+  - marketing
+  - bot
+  - traffic analysis
+  - bots
+  - fraud detection
+
+documentation:
+  - name: Installation docs
+    url: docs.newrelic.com/introduction-apm
+    description: Add APM data to use this Quickstart
+
```

Comment:
In order to install the dashboards there needs to be a data source created and assigned to this quickstart. 

https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#data-sources